src/CMakeLists.txt

```diff
@@ -67,6 +67,10 @@ else()
 	message(STATUS "Use precompiled header: OFF")
 endif(TOGGLE_PRE_COMPILED_HEADER)
 
+# AVX (Advanced Vector Extensions) support.
+if(UNIX)
+	add_compile_options(-mavx2)
+endif()
```

Comment:
@ranisalt 
yes, we are looking for a better solution, without this flag the gha compiler gave an error.

I found this solution for cmake, but I don't know if it's the best, because I don't understand anything about cmake.

https://gist.github.com/UnaNancyOwen/263c243ae1e05a2f9d0e

Reply:
Right, I see it uses 256-bit AVX registers and that requires AVX2